docs/src/modules/components/Ad.js

```diff
@@ -213,33 +193,18 @@ function Ad(props) {
     }
 
     const delay = setTimeout(() => {
-      if (!eventLabel) {
-        return;
-      }
-
       window.ga('send', {
         hitType: 'event',
         eventCategory: 'ad',
         eventAction: 'display',
         eventLabel,
       });
-
-      if (eventLabel.indexOf('in-house') === 0) {
```

Comment:
I think that you are right, we would be better merging the `ad` and `in-house` categories together. In such case, please update this line too:

https://github.com/mui-org/material-ui/blob/d1287f1874dffeade600364b36727fa7878e9a75/docs/src/modules/components/AdInHouse.js#L32

Reply:
Done